### DISCUSSION TITLE
Help! Please

### DISCUSSION THREAD

**Discussion by neujacome** [Stellarium/stellarium#220] [Q&A] [answered]
Help, because you can not see the control panels on the bottom and the left side of the program, in other versions this problem did not happen, thank you very much
Windows 10
6 GB RAM
CPU INTEL QUAD CORE Q9400 2.66 Ghz

I attached the image

![capture](https://user-images.githubusercontent.com/41518277/43037744-765ab7e4-8d11-11e8-8831-9a1c9dff285e.jpg)

**Comment by alex-w** [comment]
Please upgrade drivers for graphics card or run Stellarium in ANGLE mode.

**Comment by gzotti** [comment, marked as answer]
https://github.com/Stellarium/stellarium/wiki/FAQ#angle-mode
https://github.com/Stellarium/stellarium/wiki/FAQ#menu-invisible

**Comment by neujacome** [comment]
how do I set the ANGLE mode?

**Comment by gzotti** [comment]
By Clicking "Stellarium (ANGLE Mode)" in your start menu.
